controllers/imagelist/imagelist_controller.go

```diff
@@ -139,7 +137,7 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 
 	err = c.Watch(
 		&source.Kind{Type: &eraserv1alpha1.ImageList{}},
-		&handler.EnqueueRequestForObject{})
+		&handler.EnqueueRequestForObject{}, predicate.GenerationChangedPredicate{})
```

Comment:
So that I understand this, `predicate.GenerationChangedPredicate()` will cause the `Reconcile()` function to be run only if the ImageList's `spec` field changes, but not its status. Is that right?

Reply:
Yup that's right, status updates won't change ObjectMeta.Generation but spec changes will